packages/teleterm/src/ui/TopBar/Identity/IdentityList/IdentityList.tsx

```diff
@@ -33,7 +33,12 @@ export function IdentityList(props: IdentityListProps) {
             <Box>
               <Text bold>{props.loggedInUser.name}</Text>
               <Text typography="body2" color="text.secondary">
-                {props.loggedInUser.rolesList.join(', ')}
+                {props.loggedInUser.rolesList.map(label =>
```

Comment:
This map argument should be named `role` instead of `label` since we are mapping through a `rolesList`